### PR TITLE
build(lint): lint dist css

### LINKED ISSUE #2441: CSS Output Linting/Verification
Recently invalid CSS functions (that were actually undefined SASS functions!) slipped into our compiled component CSS.

We should add a check to lint this CSS and catch possible linting errors.

### PR DESCRIPTION
**Pull request by redallen**
Closes #2441 

- Remove unused handlebars files
- Lint CSS functions
- Disallow block comments in dist (cleans up `/* stylelint-enable */` block comments in our dist)

### CODE REVIEW

File: build/helpers/contains.js [hunk 1]
```diff
@@ -1,6 +0,0 @@
-const util = require('handlebars-utils');
```
Comment (dgutride) [inline build/helpers/contains.js]: This file is necessary for handlebars - where did the functionality get moved to?

Comment (redallen) [reply build/helpers/contains.js]: https://github.com/patternfly/patternfly-org/blob/master/packages/patternfly-4/gatsby-theme-patternfly-org/helpers/createHandlebars.js

File: gulpfile.js [hunk 2]
```diff
@@ -211,6 +201,106 @@ function buildIE() {
...
+
+  return src('./dist/**/*.css')
+    .pipe(
+      through2.obj((chunk, _, cb2) => {
+        const css = chunk.contents.toString();
+
+        let result;
+        // eslint-disable-next-line no-cond-assign
+        while ((result = regex.exec(css))) {
+          if (!validCSSFunctions.includes(result[1])) {
```
Comment (jschuler) [inline gulpfile.js]: are you ignoring case? My regex is not good enough to tell :) I think CSS functions are not case sensitive

Comment (redallen) [reply gulpfile.js]: Nice catch! I hope `node-sass`'s compiled output would never have to put this to the test, though...

**Pull request merged**